src/main/java/core/basesyntax/shop/service/Operations.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.shop.service;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public enum Operations {
+    B,
+    S,
+    P,
+    R;
```

Comment:
lets change names for BALANCE, SUPPLY... and then add them properties (ex. BALANCE("b"), SUPPLY("s"))
u need to add a filed and constructor for it